podcasts/DefaultPlayer.swift

```diff
@@ -342,7 +386,14 @@ class DefaultPlayer: PlaybackProtocol, Hashable {
         }
 
         let tapProcess: MTAudioProcessingTapProcessCallback = { tap, numberFrames, _, bufferListInOut, numberFramesOut, flagsOut in
-            var referenceToSelf = Unmanaged<DefaultPlayer>.fromOpaque(MTAudioProcessingTapGetStorage(tap)).takeUnretainedValue()
+            let referenceToSelf: DefaultPlayer
+            if FeatureFlag.useDefaultPlayerTapCookie.enabled {
```

Comment:
Can we encapsulate the code bellow to method and used on all places above? 
Something like: `getPlayerFromTap(_) -> DefaultPlayer`